variants/XMC1400/config/XMC1400_Arduino_Kit/pins_arduino.h

```diff
@@ -0,0 +1,290 @@
+#define CCU40_1_IRQn IRQ22_IRQn
+
+#define USIC0_4_IRQHandler IRQ13_Handler // I2C
+#define USIC0_4_IRQn IRQ13_IRQn
+
+#define USIC0_5_IRQHandler IRQ14_Handler // I2C
+#define USIC0_5_IRQn IRQ14_IRQn
+
+#define SCU_1_IRQHandler IRQ1_Handler //RTC
+#define SCU_1_IRQn IRQ1_IRQn
+
+#define ERU0_0_IRQHandler IRQ3_Handler // RESET
+#define ERU0_0_IRQn IRQ3_IRQn
+
+#ifdef ARDUINO_MAIN //index is arduino pin count 
+// Mapping of digital pins and comments
+const XMC_PORT_PIN_t mapping_port_pin[] =
+    {
+    /* 0  */    {XMC_GPIO_PORT1, 2}, // RX                                        P1.2
+    /* 1  */    {XMC_GPIO_PORT1, 3}, // TX                                        P1.3
+    /* 2  */    {XMC_GPIO_PORT0 , 5}, // GPIO / LED2                              P0.5
+    /* 3  */    {XMC_GPIO_PORT1 , 1}, // External int 0 / PWM40-1 output          P1.1
+    /* 4  */    {XMC_GPIO_PORT1 , 0}, // PWM40-0 output                           P1.0
+    /* 5  */    {XMC_GPIO_PORT0 , 2}, // GPIO                                     P0.2
+    /* 6  */    {XMC_GPIO_PORT0 , 6}, // PWM41-0 output                           P0.6
+    /* 7  */    {XMC_GPIO_PORT0 , 13}, // GPIO                                    P0.13
+    /* 8  */    {XMC_GPIO_PORT0 , 12}, // GPIO                                    P0.12
+    /* 9  */    {XMC_GPIO_PORT0 , 7}, // PWM40-1 output                           P0.7
+    /* 10  */   {XMC_GPIO_PORT0 , 4}, // SPI-SS                                   P0.4
+    /* 11  */   {XMC_GPIO_PORT0 , 1}, // SPI-MOSI                                 P0.1
+    /* 12  */   {XMC_GPIO_PORT0 , 0}, // SPI-MISO                                 P0.0
+    /* 13  */   {XMC_GPIO_PORT0 , 3}, // SPI-SCK                                  P0.3
+    /* 14  */   {XMC_GPIO_PORT2 , 3}, // AREF                                     P2.3 (INPUT ONLY)
+    /* 15  */   {XMC_GPIO_PORT2 , 6}, // A0 / ADC Input                           P2.6 (INPUT ONLY)
+    /* 16  */   {XMC_GPIO_PORT2 , 8}, // A1 / ADC Input                           P2.8 (INPUT ONLY)
+    /* 17  */   {XMC_GPIO_PORT2 , 9}, // A2 / ADC Input                           P2.9 (INPUT ONLY)
+    /* 18  */   {XMC_GPIO_PORT2 , 10}, // A3 / ADC Input                          P2.10
+    /* 19  */   {XMC_GPIO_PORT2 , 1}, // A4  / I2C Data / Address SDA             P2.1
+    /* 20  */   {XMC_GPIO_PORT2 , 0}, // A5  / I2C Clock SCL                      P2.0
+    /* 21  */   {XMC_GPIO_PORT2 , 4}, // RESET input ( DO NOT USE as GPIO )       P2.4 (INPUT ONLY)
+    /* 22 */    {XMC_GPIO_PORT2 , 11}, // GPIO / AD_AUX Additional Pin            P2.11
+    /* 23 */    {XMC_GPIO_PORT2 , 7}, // GPIO / AD_AUX Additional Pin             P2.7
+    /* 24 */    {XMC_GPIO_PORT2 , 5}, // GPIO / AD_AUX Additional Pin             P2.5
+    /* 25 */    {XMC_GPIO_PORT2 , 2}, // GPIO / AD_AUX Additional Pin             P2.2
+    /* 26 */    {XMC_GPIO_PORT1 , 4}, // External int 1                           P1.4
```

Comment:
NOT RESOLVED back to front modification that **BREAKS** Arduino comapatibility even within Infineon board set

Board ID is the LOWEST level of definition, not the master

Reply:
Replied below